index.html

```diff
@@ -834,34 +844,35 @@ <h4>Detecting when the next media frame will be rendered</h4>
       <h2>Recommendations</h2>
       <p>
         This section describes recommendations from the Media &amp; Entertainment
-        Interest Group for the development of a generic <a>media timed event</a> API,
-        and associated synchronization considerations.
+        Interest Group for the development of a generic <a>timed metadata</a>
+        API, and associated synchronization considerations.
       </p>
       <section>
-        <h3>Subscribing to event streams</h3>
+        <h3>Subscribing to receive timed metadata cues</h3>
         <p>
           The API should allow web applications to subscribe to receive specific
-          event streams by event type. For example, to support MPEG-DASH
+          schemas of <a>timed metadata</a> cue. For example, to support MPEG-DASH
```

Comment:
suggest s/schemas/types to be more general. Schema can be taken as a data type definition not something against which to select content.

Reply:
https://github.com/w3c/me-media-timed-events/pull/49/commits/a33e48b08e37a4d445898b5ef6b3c1f73e3b5728 replaces "schema" with "cue type"